semgrep_metrics.atd

```diff
@@ -134,6 +135,7 @@ type misc = {
     features <ocaml mutable>: string list;
     ?numFindings <ocaml mutable>: int option;
     ?numIgnored <ocaml mutable>: int option;
+    ?diffDepth <ocaml mutable>: int option;
```

Comment:
if it's a pro-engine specific thing, maybe we should define another record with pro_features: ... where
we would put this info.

Reply:
Done.